app/views/patients/_new_patient.html.erb

```diff
@@ -8,10 +8,10 @@
   <%= bootstrap_form_for patient do |f| %>
     <div class="row">
       <div class="col-sm-2">
-        <%= f.text_field :primary_phone, label: 'Phone', value: phone %>
+        <%= f.text_field :primary_phone, label: 'Phone', value: phone, required: true, pattern: '[0-9]{10}' %>
```

Comment:
I had no idea you could use a regexp pattern but that's AWESOME.

Our CMs tend to enter phones in one of three ways:

* 1112223333
* 111-222-3333
* 111.222.3333

We have a prevalidation hook in the model that strips out any non-numbers, so entry in this way is totally cool. @tingaloo could you do me a solid and adjust this regexp so that it allows those three cases?